developer/general/gsoc.md

```diff
@@ -174,45 +174,6 @@ If applicable, links to more information or discussions
 
 **Mentor**: [Frédéric Pierret](/team/)
 
```

Comment:
removed commented out section. should the information be available in some other form somewhere still?